src/js/views/Main.jsx

```diff
@@ -61,10 +61,6 @@ function Main({ user }) {
                   path="/ui/projects/import"
                   element={<Project.GitlabImport user={user} />}
                 />
-                <Route
-                  path="/ui/projects/:projectId"
-                  element={<Project.Detail user={user} />}
-                />
```

Comment:
Will `/ui/projects/1234` will be caught by the next route?

Reply:
It will, I too was dubious but the change was due to a warning being emitted by React.Router.